styles.css

```diff
@@ -363,8 +335,255 @@ p.paragraph {
 .dev-email {
   color: #42526e;
   text-decoration: none;
-  margin-top: 0.8rem;
   font-weight: 400;
   font-size: 20px;
   line-height: 32px;
+  align-items: center;
+}
+
+.mywork-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+}
+
+.about-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+}
+
+.language-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+}
+
+.socials2 {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  margin-bottom: 0.8rem;
+}
+
+.desktop-nav-container {
+  visibility: hidden;
+}
+
+.background-desktop {
+  background-image: url(./tools/header-illsutration-mobile.svg);
+  background-repeat: no-repeat;
+  background-size: initial;
+  width: 100%;
+}
+
+@media all and (min-width: 768px) {
+  .welcome {
+    width: 100%;
+    padding: 0;
+    bottom: 100px;
+    margin-top: 8rem;
+    margin-bottom: 25rem;
+  }
+
+  .background-desktop {
+    background-image: url(./tools/Header-llustration-desktop.svg);
+    background-repeat: no-repeat;
+    background-size: initial;
+    width: 100%;
+  }
+
+  .description {
+    text-align: center;
+    width: 920px;
+    height: 96px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  .body {
+    flex: none;
+    width: 100%;
+  }
+
```

Comment:
Your design looks great, you have put a lot of work into this. However, there seems to be an overflow on the page. For a better user experience on the website, kindly fix this.

![image](https://user-images.githubusercontent.com/20554583/199928790-0ad831a6-5e70-44bb-9396-b46b07ece61b.png)